src/Expander/Syntax.hs

```diff
@@ -8,6 +8,7 @@
 module Expander.Syntax where
 
 import Control.Monad.Except
+import Control.Monad.IO.Class
```

Comment:
unused import?

Reply:
No this depends on the version of mtl. I had to add it for 40d4fccd20e4e43123cb415b64c4b251239022c3. On master we pull in mtl-2.2.2 so `Control.Monad.Except` re-exports `Control.Monad.Trans` which re-exports `Control.Monad.IO.Class`. But in order to use `WriterT.CPS` we need mtl-2.3 which does not do these re-exports.

So basically this is a separate issue. IMHO we should drop support for GHC-8.8 and below. Leaving 8.10.7 as the oldest supported GHC. This would also cleanup all the warnings around the `Pretty` library.

Reply:
Let's drop everything under 8.10 :-)